### PR TITLE
Updates for Netlibs 3.12.0 and ROCM 6.0

### PR DESCRIPTION
**Pull request by ItsPitt**

### CODE REVIEW

File: src/base/flamec/wrappers/blas/3/hip/FLA_Trmm_external_hip.c [hunk 1]
```diff
@@ -90,7 +90,8 @@ FLA_Error FLA_Trmm_external_hip( rocblas_handle handle, FLA_Side side, FLA_Uplo
                    n_B,
                    buff_alpha,
                    ( float * ) A_mat, ldim_A,
-                   ( float * ) B_mat, ldim_B );
+                   ( float * ) B_mat, ldim_B,
+                   ( float * ) B_mat, ldim_B);
```
Comment (ItsPitt) [inline src/base/flamec/wrappers/blas/3/hip/FLA_Trmm_external_hip.c:94]: Rocm 6.0 fixes

File: netlib-test/create_new_testdir.sh [hunk 2]
```diff
@@ -131,6 +131,9 @@ main()
 	sed -i 's/zchkaa.o: zchkaa.F/zchkaa.o: zchkaa.F $(LAPACKLIB) $(BLASLIB)/' ${testdir_new}/TESTING/LIN/Makefile
 	sed -i 's/$(FC) $(FFLAGS_DRV) -c -o $@ $</$(FC) $(FFLAGS_DRV) -c -o $@ $^/g' ${testdir_new}/TESTING/LIN/Makefile
 
+	# A small fix so this script will parse correctly
+	sed -i 's/find(\"run/find(\"run\)/g' ./${testdir_new}/lapack_testing.py
+
```
Comment (ItsPitt) [inline netlib-test/create_new_testdir.sh:138]: A small charge to the script. It was erroring out after all the tests had been run and it was trying to collect the data

Comment (iotamudelta) [reply netlib-test/create_new_testdir.sh:138]: can you just add that into the comment in the bash script? as in "do $this b/c of $that"? Sth like "script errors out after all tests completed at data collection. Correct find path." (assuming that is correct).

**Review by iotamudelta — CHANGES_REQUESTED** [review]
Small change to the documentation.

File: netlib-test/create_new_testdir.sh (hunk 2 above)
Comment (ItsPitt) [reply netlib-test/create_new_testdir.sh:138]: Fixed in https://github.com/iotamudelta/libflame/pull/10/commits/0c0c07281e4b5c4d9f82bfe6d5fb17ab0b84f2ce

**Review by iotamudelta — APPROVED** [review]
LGTM

**Pull request merged**